Dockerfile

```diff
@@ -1,9 +1,9 @@
-FROM node:8.12.0-alpine
+FROM node:12.14-alpine3.9
 EXPOSE 3000
 COPY . /app
 RUN apk update && \
     apk upgrade && \
-    apk add git libgit2-dev && \
+    apk add git libgit2-dev krb5-libs && \
```

Comment:
why is krb5-libs needed?

Reply:
Because the file `libgssapi_krb5.so` is needed by _nodegit_.
This dependency is undocumented by _nodegit_ but usually _krb5-libs_ is already installed on GNU/Linux desktop distributions.

The following **build error** is throw without _krb5-libs_:

> [nodegit] WARN - Could not finish postinstall
Error: Command failed: node "/app/node_modules/nodegit/dist/nodegit.js"
/app/node_modules/nodegit/dist/nodegit.js:16
    throw ex;
    ^
Error: Error loading shared library libgssapi_krb5.so.2: No such file or directory (needed by /app/node_modules/nodegit/build/Release/nodegit.node)
    at Object.Module._extensions..node (internal/modules/cjs/loader.js:1021:18)
    at Module.load (internal/modules/cjs/loader.js:811:32)
    at Function.Module._load (internal/modules/cjs/loader.js:723:14)
    at Module.require (internal/modules/cjs/loader.js:848:19)
    at require (internal/modules/cjs/helpers.js:74:18)
    at Object.<anonymous> (/app/node_modules/nodegit/dist/nodegit.js:12:12)
    at Module._compile (internal/modules/cjs/loader.js:955:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:991:10)
    at Module.load (internal/modules/cjs/loader.js:811:32)
    at Function.Module._load (internal/modules/cjs/loader.js:723:14)
    at ChildProcess.exithandler (child_process.js:295:12)
    at ChildProcess.emit (events.js:223:5)
    at maybeClose (internal/child_process.js:1021:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:283:5) {
  killed: false,
  code: 1,
  signal: null,
  cmd: 'node "/app/node_modules/nodegit/dist/nodegit.js"'
}


and the following **runtime error** is throw without _krb5-libs_:

> Error: Error loading shared library libgssapi_krb5.so.2: No such file or directory (needed by /app/node_modules/nodegit/build/Release/nodegit.node)
    at Object.Module._extensions..node (internal/modules/cjs/loader.js:1021:18)
    at Module.load (internal/modules/cjs/loader.js:811:32)
    at Function.Module._load (internal/modules/cjs/loader.js:723:14)
    at Module.require (internal/modules/cjs/loader.js:848:19)
    at require (internal/modules/cjs/helpers.js:74:18)
    at Object.<anonymous> (/app/node_modules/nodegit/dist/nodegit.js:12:12)
    at Module._compile (internal/modules/cjs/loader.js:955:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:991:10)
    at Module.load (internal/modules/cjs/loader.js:811:32)
    at Function.Module._load (internal/modules/cjs/loader.js:723:14)